giskard/llm/client/copilot.py

```diff
@@ -0,0 +1,95 @@
+from typing import Optional, Sequence
+
+from dataclasses import dataclass
+from logging import warning
+
+from ..config import LLMConfigurationError
+from ..errors import LLMImportError
+from .base import ChatMessage
+from .openai import AUTH_ERROR_MESSAGE, OpenAIClient
+
+try:
+    import openai
+except ImportError as err:
+    raise LLMImportError(flavor="talk") from err
+
+
+@dataclass
+class ToolChatMessage(ChatMessage):
+    name: Optional[str] = None
+    tool_call_id: Optional[str] = None
+    tool_calls: Optional[list] = None
+
+
+def _format_message(msg: ChatMessage) -> dict:
```

Comment:
should be `ToolChatMessage` type

Reply:
actually for this one, it's `ChatMessage`, but what `GiskardCopilotClient.complete` returns is `ToolChatMessage`